src/ape_test/accounts.py

```diff
@@ -6,15 +6,17 @@
 
 from ape.api import TestAccountAPI, TestAccountContainerAPI, TransactionAPI
 from ape.types import AddressType, MessageSignature, TransactionSignature
-from ape.utils import GeneratedDevAccount, cached_property, generate_dev_accounts
+from ape.utils import GeneratedDevAccount, generate_dev_accounts
 
 
 class TestAccountContainer(TestAccountContainerAPI):
+    _num_generated = 0
+
     @property
     def config(self):
         return self.config_manager.get_config("test")
 
-    @cached_property
+    @property
```

Comment:
This could probably stay cached, since config shouldn't be changing

Reply:
Okay, following up from discord, this causes a failure when it's cached due to how we are testing ape

Oppurtunity for future optimization